tox.ini

```diff
@@ -1,6 +1,9 @@
 [tox]
 envlist = py27, py34, py35
 
+[testenv:docs]
+commands = python setup.py build_sphinx
```

Comment:
so we generate documents when we run test? if that is the case, it does not make sense. can we add a script in scripts folder to generate docs. If we want to add the generation here, we need to make sure it fails if there is a difference.